tests/functests/instancetype_test.go

```diff
@@ -253,7 +253,16 @@ func getClusterPreferences(virtClient kubecli.KubevirtClient) []instancetypev1be
 	clusterPreferences, err := virtClient.VirtualMachineClusterPreference().List(context.Background(), metav1.ListOptions{})
 	Expect(err).ToNot(HaveOccurred())
 	Expect(clusterPreferences.Items).ToNot(BeEmpty())
-	return clusterPreferences.Items
+
+	// Some preferences may not have spec.Requirements, these should be skipped from testing.
```

Comment:
Can you change the name of this helper to something like `getClusterPreferencesWithRequirements` or `clusterPreferencesWithRequirements` without the `get`?

Can you also move this into the "VirtualMachine using a preference with resource requirements" `Context` so it's alongside its only callers?

Reply:
Sure! Done.